Tests.Integration.Presentation.Web/Contract/ItContractTest.cs

```diff
@@ -94,6 +94,44 @@ public async Task Cannot_Add_PaymentMilestone(OrganizationRole role)
             }
         }
 
+        [Theory]
+        [InlineData(OrganizationRole.GlobalAdmin)]
+        [InlineData(OrganizationRole.LocalAdmin)]
+        public async Task Can_Add_CriticalityType(OrganizationRole role)
+        {
+            //Arrange
+            var login = await HttpApi.GetCookieAsync(role);
+            var contract = await ItContractHelper.CreateContract(A<string>(), OrganizationId);
+            var critircalityName = A<string>();
+            var criticality = await EntityOptionHelper.CreateOptionTypeAsync(EntityOptionHelper.ResourceNames.CriticalityTypes, critircalityName, OrganizationId);
+
+            //Act - perform the action with the actual role
+            await ItContractHelper.AssignCriticalityTypeAsync(contract.OrganizationId, contract.Id, criticality.Id, login);
+            var contractResult = await ItContractHelper.GetItContract(contract.Id);
+
+            //Assert
+            Assert.Equal(contract.Id, contractResult.Id);
+            Assert.Equal(criticality.Id, contractResult.CriticalityTypeId);
+            Assert.Equal(criticality.Name, contractResult.CriticalityTypeName);
+        }
+
+        [Theory]
+        [InlineData(OrganizationRole.User)]
+        public async Task Cannot_Add_CriticalityType(OrganizationRole role)
+        {
+            //Arrange
+            var login = await HttpApi.GetCookieAsync(role);
+            var contract = await ItContractHelper.CreateContract(A<string>(), OrganizationId);
+            var criticalityName = A<string>();
+            var criticality = await EntityOptionHelper.CreateOptionTypeAsync(EntityOptionHelper.ResourceNames.CriticalityTypes, criticalityName, OrganizationId);
```

Comment:
why create a new option type in this test... is it not enough just to query the existing and pick a random one?